packages/terra-clinical-result/docs/ClinicalResultBloodPressure.md

```diff
@@ -0,0 +1,95 @@
+# Terra Clinical Result Blood Pressure
+
+The Clinical Result is a starndardized view for single result values added to a Patient's Medical Chart. This is specifically for Blood Pressure Vital Sign results.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+```
+$ npm install terra-clinical-result
+```
+
+<!-- AUTO-GENERATED-CONTENT:START Peer Dependencies -->
+<!-- AUTO-GENERATED-CONTENT:END -->
+
+## Component Features
+
+* [Cross-Browser Support](https://github.com/cerner/terra-ui/blob/master/src/terra-dev-site/contributing/ComponentStandards.e.contributing.md#cross-browser-support)
+* [Responsive Support](https://github.com/cerner/terra-ui/blob/master/src/terra-dev-site/contributing/ComponentStandards.e.contributing.md#responsive-support)
+* [Mobile Support](https://github.com/cerner/terra-ui/blob/master/src/terra-dev-site/contributing/ComponentStandards.e.contributing.md#mobile-support)
+* [LTR/RTL Support](https://github.com/cerner/terra-ui/blob/master/src/terra-dev-site/contributing/ComponentStandards.e.contributing.md#ltr--rtl-support)
+ 
+<!-- Uncomment supported features.
+* [Internationalization Support](https://github.com/cerner/terra-ui/blob/master/src/terra-dev-site/contributing/ComponentStandards.e.contributing.md#internationalization-i18n-support)
+-->
+
+## Usage Guide
+
+The blood pressure result is constructed similarly to the single clinical result by providing structured object with a specific construction that follows a similar pattern to the [HL7 FHIR Observation Blood Pressure](https://www.hl7.org/fhir/observation-example-bloodpressure.html) example.
```

Comment:
Lets move this to a real example that users can toggle the src code. We've been moving toward the usage section simply showing the component import(s) for quick ref